forward-message/assets/ce-paste-theme.css

```diff
@@ -0,0 +1,189 @@
+body {
+  margin: 0;
+  font-family: "Inter var experimental", "Inter var", -apple-system,
+    BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell,
+    "Helvetica Neue", sans-serif;
+  font-size: 16px;
+  line-height: 1.5rem;
+  color: rgb(18, 28, 45);
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+a {
+  color: rgb(2, 99, 224);
+}
+
+a:visited {
+  color: rgb(2, 99, 224);
+}
+
+ul {
+  padding: 0;
+  list-style-type: none;
+}
+
+ol.steps {
+  counter-reset: step;
+  list-style-type: none;
+}
+
+ol.steps > li::before {
+  counter-increment: step;
+  content: "Step " counter(step) ": ";
+  font-weight: 700;
+}
+
+header {
+  display: flex;
+  color: white;
+  background: rgb(6, 3, 58);
+  width: 100%;
+  box-shadow: 0px 2px 6px 0px rgba(75, 86, 113, 0.2);
+  align-content: center;
+}
+
+header > nav {
+  display: flex;
+  flex-direction: column;
+  padding-left: 2rem;
+  border-left: 1px solid rgba(255, 255, 255, 0.2);
+  padding-top: 0.75rem;
+  padding-bottom: 0.75rem;
+}
+
+header > nav span {
+  font-weight: 400;
+}
+
+header > nav aside {
+  color: rgb(200, 175, 240);
+  font-size: 0.75rem;
+}
+
+header > nav svg {
+  width: 1rem;
+  height: 1rem;
+}
+
+main {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  font-size: 0.875rem;
+}
+
+div.page-top {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+div.content {
+  display: flex;
+  flex-direction: column;
+  width: 75%;
```

Comment:
if you remove the `width: 75` you'll get the content more evenly centered on the page

Reply:
Good catch! I changed this to `max-width: 75%` so it behaves a bit better at mobile screen sizes; take a look and let me know if it seems ok!